src/Workspaces/Remote/ServiceHub/Services/Renamer/RemoteRenamerService.cs

```diff
@@ -103,12 +105,13 @@ private CodeCleanupOptionsProvider GetClientOptionsProvider(RemoteServiceCallbac
                 if (symbol is null)
                     return null;
 
-                var locations = await LightweightRenameLocations.TryRehydrateAsync(
-                    solution, GetClientOptionsProvider(callbackId), serializableLocations, cancellationToken).ConfigureAwait(false);
-                if (locations == null)
```

Comment:
similarly, we should not be going through LightweightRenameLocations on the oop side.